### PR TITLE
Pagination for sourcing org users and nominations view refactor and removed list nominations component as it was not used anywhere

### PR DESCRIPTION
**Pull request by twsvaishali**

### CODE REVIEW

File: src/app/client/src/app/modules/cbse-program/components/collection/collection.component.html [hunk 1]
```diff
@@ -8,14 +8,19 @@ <h5 class="m-0 p-0 font-weight-bold d-inline-block">{{programContext.name}} </h5
           <ng-template #popupTemplate>
             <div class="content">
               <div class="textbook-details ml-8">
+                <div class="header"><strong>{{ resourceService.frmelmnts.lbl.mediums }}</strong></div>
```
Comment (twsvaishali) [inline src/app/client/src/app/modules/cbse-program/components/collection/collection.component.html:11]: @pallakartheekreddy @venkateshwarans  Changes are for the issue https://project-sunbird.atlassian.net/browse/DP-580 -

File: src/app/client/src/app/modules/cbse-program/components/collection/collection.component.html [hunk 2]
```diff
@@ -232,116 +237,6 @@ <h4 class="sb-pageSection-title six">{{resourceService.frmelmnts.lbl.textbooks}}
       </div>
     </div>
   </div>
-
-
-
-  <div class="ui container pb-48 hidden hide">
```
Comment (twsvaishali) [inline src/app/client/src/app/modules/cbse-program/components/collection/collection.component.html:238]: @pallakartheekreddy @venkateshwarans  this was having class "hidden hide", never used

Comment (venkateshwarans) [reply src/app/client/src/app/modules/cbse-program/components/collection/collection.component.html:238]: Yeah. I noticed it too. I removed this long back. But somehow sneaked back in.

File: src/app/client/src/app/modules/program/components/program-nominations/program-nominations.component.html [hunk 3]
```diff
@@ -126,8 +111,12 @@ <h5 class="m-0 p-0 font-weight-bold d-inline-block">{{ programDetails?.name }} <
                 </div>
             </div>
             <div class="ui segment p-0 m-0 mt-4 sb-bg-color-white dynamicHeight" suiTabContent="2">
-                <div *ngIf="programCollections && programCollections.length && contentAggregationData && !isEmptyObject(programContext)" class="ui container p-0 m-0">
-                    <app-textbook-list [programDetails]="programContext" [collectionsInput]="programCollections" [contentAggregationInput]="contentAggregationData" (selectedCollection)="viewContribution($event)"></app-textbook-list>
+                <app-loader *ngIf='showNominationLoader'></app-loader>
+                <div *ngIf="!showTextbookLoader && programCollections && programCollections.length && contentAggregationData && !isEmptyObject(programDetails)" class="ui container p-0 m-0">
```
Comment (venkateshwarans) [inline src/app/client/src/app/modules/program/components/program-nominations/program-nominations.component.html:115]: showTextbookLoader or showNominationLoader?

**Pull request merged**